packages/roosterjs-editor-plugins/lib/plugins/TableResize/TableResize.ts

```diff
@@ -335,6 +337,14 @@ export default class TableResize implements EditorPlugin {
 
     private startVerticalResizeTable = (e: MouseEvent) => {
         this.resizingState = ResizeState.Vertical;
+
+        const vtable = new VTable(this.currentTd);
+        if (vtable) {
+            const rect = normalizeRect(this.currentTd.getBoundingClientRect());
+            this.currentCellsToResize = vtable.getCellsWithBorder(rect.right, true);
+            this.nextCellsToResize = vtable.getCellsWithBorder(rect.right, false);
```

Comment:
Any special handling for RTL?

Reply:
I don't think we need special handling for RTL as it is working regardless of LTR or RTL

Reply:
I found that in RTL the whole resizing functionality not working correctly. This also repro without this change. I remember it worked before, not sure which change broke it.

I suggest we fix the RTL issue first then do this change.

![image](https://user-images.githubusercontent.com/23065085/113364710-0d2d6680-9309-11eb-9ecc-3cf231ff9488.png)

Reply:
fixed